client/payment-gateways/disable-confirmation-modal.js

```diff
@@ -86,11 +99,15 @@ const DisableConfirmationModal = ( { onClose, onConfirm } ) => {
 			</ul>
 			<p>
 				{ interpolateComponents( {
-					mixedString: __(
-						'{{strong}}Need help?{{/strong}} ' +
-							'Learn more about {{wooCommercePaymentsLink}}WooCommerce Payments{{/wooCommercePaymentsLink}} or ' +
-							'{{contactSupportLink}}contact WooCommerce Support{{/contactSupportLink}}.',
-						'woocommerce-payments'
+					mixedString: sprintf(
+						/* translators: %s: WooPayments */
+						__(
+							'{{strong}}Need help?{{/strong}} ' +
+								'Learn more about {{wooCommercePaymentsLink}}%s{{/wooCommercePaymentsLink}} or ' +
```

Comment:
Nitpick: rename component to `wooPaymentsLink`

Reply:
sorry Vlad, we agreed not to touch the code, I'll leave it as-is for now as it's not user-facing
next time!

Reply:
No worries. Thanks for letting me know.